improver_tests/acceptance/SHA256SUMS

```diff
@@ -76,8 +76,8 @@ ba16a5cd5174bcddd08cad6509b38e7a221e18073fe09f41400a911937446b20  ./construct-re
 350a568d28eab64886ccaca8f057745c98a3fc91e9861dff3e5da1bba24574c8  ./construct-reliability-tables/basic/kgo_without_single_value_bins.nc
 9480392febfe81b76bf93de6ca2d88e07ad3fccabbca343713ef7ee374a77d47  ./construct-reliability-tables/basic/truth_0.nc
 d15752239edb3fb26904b600f1705ea97ac8c637090033f16ddeb86052eed5b4  ./construct-reliability-tables/basic/truth_1.nc
-c96b835c7c5888ca42ce94e0b0ba85a98ffdeb5baf1f12b794f1155ee024bd76  ./convection-ratio/basic/convective_precipitation_rate.nc
-534a0b49e6ede4c913c730045f3b17fca59b4093278d3c84014f730893e1a71b  ./convection-ratio/basic/dynamic_precipitation_rate.nc
+74f850942572aa99de807396d48bd80dd96088c638a9d5fa379b95f7c5ad8614  ./convection-ratio/basic/convective_precipitation_rate.nc
+b946c7687cb9ed02a12a934429a31306004ad45214cf4b451468b077018c0911  ./convection-ratio/basic/dynamic_precipitation_rate.nc
```

Comment:
Perhaps it would be slightly preferable to rename these files, inline with the changes made elsewhere in this PR. The primary benefit would be to use `stratiform`  consistently in favour of `dynamic`. This would need an associated change in `test_convection_ratio.py`.